firebase-app-distribution/src/test/java/com/google/firebase/appdistribution/FirebaseAppDistributionTest.java

```diff
@@ -87,9 +94,38 @@ public void setup() {
                 .setApiKey(TEST_API_KEY)
                 .build());
 
-    firebaseAppDistribution = new FirebaseAppDistribution(firebaseApp, mockFirebaseInstallations);
+    firebaseAppDistribution =
+        new FirebaseAppDistribution(
+            firebaseApp, mockFirebaseInstallations, mockFirebaseAppDistributionTesterApiClient);
 
     when(mockFirebaseInstallations.getId()).thenReturn(Tasks.forResult(TEST_FID_1));
+    when(mockFirebaseInstallations.getToken(true))
+        .thenReturn(Tasks.forResult(mockInstallationTokenResult));
+    when(mockInstallationTokenResult.getToken()).thenReturn(TEST_AUTH_TOKEN);
+
```

Comment:
This is a nit but you could make this private static variables (TEST_RELEASE_1 and TEST_RELEASE_2). That way, down below, instead of doing `assertEquals("3", release.getBuildVersion());` you could do `assertEquals(TESTER_RELEASE_1.getBuildVersion(), release.getBuildVersion()`. 

This is generally a good pattern to follow to using string constants in place (like the "3" above). It makes the code more readable